config-connector/solutions/apps/helm/wordpress/README.md

```diff
@@ -0,0 +1,55 @@
+# Wordpress
+
+==================================================
+
+## NAME
+
+  wordpress
+
+## SYNOPSIS
+
+The WordPress application demonstrates how you can configure a WordPress site powered by GCP MySQL database and using Workload Identity for authentication.
+
+## CONSUMPTION
+
+  ```bash
+   helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+  ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+1. [Helm](https://helm.sh/docs/using_helm/)
+
+## USAGE
+
+All steps are run from this directory.
+
+1. Review and update the values in `./charts/wordpress-gcp/values.yaml` .
+1. Validate and install the sample with Helm
+
+    ```bash
+    # validate your chart
+    helm lint ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+
+    # check the output of your chart
+    helm template ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID]
+
+    # install your chart
+    helm install ./charts/wordpress-gcp/ --set google.projectId=[PROJECT_ID] --generate-name
+    ```
+
+1. Check the status of your database by running `kubectl describe sqlinstance wp-db`. Once the database is created, obtain the external IP address of your WordPress application by checking `kubectl get svc wordpress-external`. Navigate to this address and validate that you see WordPress installation page.
```

Comment:
Received this error `Error 403: The client is not authorized to make this request., notAuthorized` when running `kubectl describe sqlinstance wp-db`. Seems that we need some additional permissions to make it work.

Reply:
Added sqladmin serviceusage api. Trying with that...

Reply:
Added serviceusage and also additional instructions.

Reply:
Verified this is addressed.